chaos-mesh/chaos-mesh
# Use bigheap to inject heap stress

Signed-off-by: Yang Keao <keao.yang@yahoo.com>

### What problem does this PR solve?

Fix #737 

### What is changed and how does it work?

Use `--bigheap` to inject memory stress rather than `--vm` which is virtual memory.

### Check List


Tests


- Manual test

Code changes

- Has Go code change

Side effects

- Breaking backward compatibility

Remove `Size` field of memory stressor, because we cannot set the total size of `bigheap` stressor, only the speed is configurable.

Maybe we should provide some options to choose the memory stressors between `brk`, `bigheap` and `stack`, do you think it is needed?
